.github/workflows/build-to-release-branches.yml

```diff
@@ -0,0 +1,25 @@
+name: Build to release branches
+
+on:
+  workflow_dispatch:
+  push:
+    tags: ['*']
```

Comment:
Is not the `master` here missing for updating `master-built` on push?

Reply:
I added another comment and updated the PR, waiting for your review @Chrico